api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -505,10 +506,11 @@ public GenomicExtractionJob submitGenomicExtractionJob(
             .get()
             .createSubmission(
                 new FirecloudSubmissionRequest()
-                    .deleteIntermediateOutputFiles(true)
                     .methodConfigurationNamespace(methodConfig.getNamespace())
                     .methodConfigurationName(methodConfig.getName())
-                    .useCallCache(false),
+                    .deleteIntermediateOutputFiles(true)
+                    .useCallCache(true)
+                    .memoryRetryMultiplier(BigDecimal.valueOf(1.5)),
```

Comment:
this could be a const, but 1.5 is also what the value *means* (try again with 1.5x memory) so I'm inclined not to make it one